### DISCUSSION TITLE
EMAIL_ADDRESS event improvements

### DISCUSSION THREAD

**Discussion by domwhewell-sage** [blacklanternsecurity/bbot#925] [Ideas]
Could we look into a module for linkedIn scraping? 

Also would it be possible to output all `EMAIL_ADDRESS` events in their own txt file? This could be useful information for potential targets of social engineering etc.

**Comment by TheTechromancer** [comment]
Yes it could probably be an output module; we already have something similar for subdomains.

As for linkedin scraping, it's a tough thing to approach because linkedin is always working so hard to prevent it. In my personal experience, if you're looking for names and job titles, you're better off scraping google/ddg/bing search results. But it's been a while since I looked into this so it's possible there's a better way. I'm open to suggestions!

**Reply by domwhewell-sage** [reply]
Yes that's probably what I was thinking of scraping social accounts from google/ddg/bing search results.
The only other way I know is using the LinkedIn Sales Navigator but that tool comes with a hefty price tag so probably best to stick to scraping search results.
